app/src/main/AndroidManifest.xml

```diff
@@ -41,7 +41,7 @@
         <!-- For ADM -->
         <amazon:enable-feature
             android:name="com.amazon.device.messaging"
-            android:required="true"/>
+            android:required="false"/>
```

Comment:
Can you add a comment to explain what this does? e.g. "if false, graceful degradation requried. if true, (and I'm not sure if this is true), APK will not be available to devices without ADM."